cmd/open_other.go

```diff
@@ -12,19 +12,33 @@ import (
 
 var tpmPath string
 
+// tpmWrapper wrap tpm io.ReadWriteCloser and EventLogGetter interfaces together.
```

Comment:
tpmWrapper wraps a TPM io.ReadWriteCloser that implements client.EventLogGetter.